src/main/java/io/pinecone/clients/Pinecone.java

```diff
@@ -46,6 +49,10 @@ public class Pinecone {
         this.manageIndexesApi = manageIndexesApi;
     }
 
+    public PineconeConfig getConfig() {
```

Comment:
Added getConfig for testing purpose

Reply:
If just for testing suggest making it package-private